src/actions/account.js

```diff
@@ -145,11 +99,13 @@ export const login = ({ passphrase, publicKey, hwInfo }) => async (dispatch, get
   const { network, settings } = getState();
   dispatch(accountLoading());
 
-  const activeTokens = Object.keys(settings.token.list)
-    .filter(key => settings.token.list[key]);
-  const [error, info] = await to(getAccounts(activeTokens, {
-    network, publicKey, passphrase,
-  }));
+  const params = Object.keys(settings.token.list)
+    .filter(key => settings.token.list[key])
```

Comment:
Are you filtering something here? Should this be a `map` may be?

Reply:
yes, the value of `settings.token.list[key]` is a boolean.